src/content/app/genome-browser/hooks/useGenomicTracks.ts

```diff
@@ -66,6 +67,11 @@ const useGenomicTracks = () => {
     );
     trackIdsList.forEach(genomeBrowserMethods.toggleTrack);
     genomeIdInitialisedRef.current = activeGenomeId as string;
+
+    // reset this ref on unmount to rerun this useEffect on mount for the last selected genome id
+    return () => {
+      genomeIdInitialisedRef.current = '';
+    };
```

Comment:
I am awfully confused by this. The cleanup function from a useEffect runs when a component is unmounted; but when it is unmounted, the ref should disappear as well. Refs don't survive after a component is removed.

Reply:
It doesn't seem to be resetting by itself once I remove that function.

![Screenshot 2022-11-25 at 17 18 36](https://user-images.githubusercontent.com/32512909/204032180-536176f7-5f5d-41b8-b89e-95e7abd4db34.png)